contracts/src/v0.8/interfaces/AuthorizedReceiverInterface.sol

```diff
@@ -0,0 +1,10 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.19;
```

Comment:
not clear on this one either 😅 could you move this to the operatorforwarder/interfaces folder

Reply:
On that, I can abs move it but it's actually a repetitive pattern I've seen across the repo that folks reimplement this same logic from scratch in their products, does it make sense to keep it here for future development to base on it? happy to move as well